apps/casdoor/1.780.0/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   casdoor:
-    image: casbin/casdoor:v1.780.0
+    image: casbin/casdoor:v1.781.0
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The code appears to be correct and there aren't any issues, irregularities, or optimizations required at this time. Just make sure that the `${CONTAINER_NAME}` environment variable is properly defined in your deployment configuration for `casdoor`. If you have further specific requirements or need additional modifications, please let me know!